app/assets/stylesheets/_date-picker.scss

```diff
@@ -0,0 +1,30 @@
+.react-datepicker {
```

Comment:
I think it's a good one, but since we're using SASS we can make it even better without changing how it works by using SASS' `&`.